### ISSUE TITLE
Create a pseudolocale for i18n testing purposes

### ISSUE THREAD

**Issue by ray-kraesig** [zulip/zulip-mobile#4052]
For context: [Wikipedia's article on pseudolocalization](https://en.wikipedia.org/wiki/Pseudolocalization).

Unfortunately, we don't do much in-house testing of our localization, and this can lead to issues like #4013. This could be ameliorated somewhat if we were to autogenerate translations for pseudolocale from `messages_en.json`, and encourage any (mostly-)monolingual Anglophones to test from within it. Any unlocalized strings would stand out as being undecorated.

**Comment by agrawal-d** [comment]
This is done in the Zulip/Zulip repository. 
In their tests, i18n is emulated as:

Original String: "Uploading filename..."
String that appears in tests: "translated: Uploading filename..."

This was helpful, because while working on my PR and writing tests, I realised some of my code was incorrect as it left some strings un-localized.

Is above an example of pseudo-localization?

**Comment by ray-kraesig** [comment]
> Is above an example of pseudo-localization?

Yes – and it's also an example of pseudolocalization being useful!
